### PR TITLE
Changes to Deer mount v0.1

### PR DESCRIPTION
**Pull request by Chreatures**
From the poll on discord, i propose some changes to the deer mount:
To diversify between the growth stages and add wisdom to the older deer with vines (vines are also on medium antlers, as not every deer reaches the majestic stadium)

Antlers:
- Small, changed: +1D5 DEX -> +1D3 DEX
- Medium, added: +1D3 DEX , +1 STR
- Majestic, changed: +1D5 DEX, +1D5 STR -> +1D3 DEX, +1D3 STR
- Antlers with vines, added: +1WIS

Deer with a slime attached can be unresponsive sometimes, but can also bring unexpected things! The purple bandana has an orange glowing stone attached, and the red bandana has a blue glowing stone attached

Accessories:
- Slime, added: -1D3 DEX +1D3 LUC
- Purple Bandana, added: +1 CON
- Red Bandana, added: +1 INT

Some deer are kept very clean (flashy hooves), some are charmy (flashy eyes) and some have uncommon colors (emerald and rudolf nose), those traits could be liked by some and give a certain charm to the owner

Eye, Hooves:
- one of the two "flashy", added: 20% +1CHA
- two with "flashy", added: 50% +1CHA
- emerald or rudolf nose, added: 50% +1CHA

### CODE REVIEW

File: attribute_assign.py [hunk 1]
```diff
@@ -90,9 +90,26 @@ def assign_attribute(stats):
...
+        stats.dex = stats.dex - random.randint(1, 3)
+        stats.luc = stats.luc + random.randint(1, 3)
+    if stats.mount in Bandana_purple_Deer_list:
+        stats.con = stats.con + 1
+    if stats.mount in Bandana_red_Deer_list:
+        stats.int = stats.int + random.randint(1, 2)
+    if stats.mount in One_Blink_Deer_list:
+        stats.cha = stats.cha + random.randint(0, 1)
+    if stats.mount in Two_Blink_Deer_list:
+        stats.cha = stats.cha + stats.cha + 1
```
Comment (chiaslimes) [inline attribute_assign.py]: I think this might be doubling CHA

Comment (Chreatures) [reply attribute_assign.py]: No deer in One_Blink_Deer_List are also on Two_Blink_Deer_List if it's that what you mean. It's very possible that i used some formula in the wrong way

Comment (chiaslimes) [reply attribute_assign.py]: Yeah so this line 110: 
stats.cha = stats.cha + stats.cha + 1

So if CHA is 12 then;
12 + 12 + 1 = 25

Can probably just remove one of the stats.cha after the equals sign :)

Comment (Chreatures) [reply attribute_assign.py]: Should i do the same on all the lines that use this formula? LIke 110 114 118

Comment (Chreatures) [reply attribute_assign.py]: Wait is it already edited? Because I was confused not seeing it in the files now

Comment (chiaslimes) [reply attribute_assign.py]: All good! looks like your file replacement commit has this covered :)

**Pull request merged**